src/api/handlers/webhook.ts

```diff
@@ -270,6 +271,42 @@ export function upsertSlashCommand(
   return result;
 }
 
+/**
+ * Bulk edit existing slash commands. If a command does not exist, it will create it.
+ * 
+ * **NOTE:** Any slash commands that are not specified in this function will be **deleted**. If you don't provide the commandID and rename your command, the command gets a new ID.
+ */
+export async function upsertSlashCommands(
+  options: UpsertSlashCommandsOptions[],
+  guildID?: string,
+) {
+  const data = options.map((option) => {
+    // Use ... for content length due to unicode characters and js .length handling
+    if ([...option.name].length < 2 || [...option.name].length > 32) {
+      throw new Error(Errors.INVALID_SLASH_NAME);
+    }
+
+    if (
+      [...option.description].length < 1 || [...option.description].length > 100
+    ) {
+      throw new Error(Errors.INVALID_SLASH_DESCRIPTION);
+    }
+
+    option["id"] = option.commandID;
```

Comment:
What is this for?

Reply:
The param discord takes for the commandID is just named id for the request so it needs to be set here.